workflows/pgbench/main.workflow

```diff
@@ -0,0 +1,20 @@
+workflow "Run PGBenchmarks" {
+  resolves = "analyse"
+}
+action "lint" {
+  uses = "actions/bin/shellcheck@master"
+  args = "-x ./workflows/pgbench/benchmark.sh ./workflows/pgbench/analyze.sh"
+}
+action "benchmark" {
+  needs = "lint"
+  uses = "actions/docker/cli@master"
+  runs = ["./workflows/pgbench/benchmark.sh"]
+  env = {
+    PG_IMAGES = "pg9.6,pg9.3"
```

Comment:
can we change this to `PG_IMAGES`? In this way, we get the full image name and can test agains any arbitrary image, not only the ones from the official repo.

Reply:
you mean pg9.6 to be postgres:9.6, right? or is it something I am not getting?

Reply:
sorry, yes, I meant that we can assume the full name of the image is provided, instead of the tags. In this way the logic of the `benchmark.sh` can be simplified, and we allow any arbitrary postgres image to be tested